lintro/ai/review/error_display.py

```
@@ -0,0 +1,151 @@
+    output.print()
+    output.print(
+        Panel(
+            text,
+            title=f"[bold red]Review failed[/bold red] — {title}",
+            border_style="red",
+            padding=(1, 2),
+        ),
+    )
+
+
+def _format_error(error: AIError | ValueError) -> tuple[str, str, list[str]]:
```

Ruff (0.15.20)

[warning] Too many return statements (7 > 6)

(PLR0911)